Makefile

```diff
@@ -2,39 +2,44 @@
 -include build/private/bgo_exports.makefile
 include ${BGO_MAKEFILE}
 
-export CGO_ENABLED=0
-export T=github.com/aws/amazon-eks-pod-identity-webhook
-UNAME_S = $(shell uname -s)
-GO_LDFLAGS = -ldflags='-s -w -buildid=""'
-
 install:: build
-ifeq ($(UNAME_S), Darwin)
-	GOOS=darwin GOARCH=amd64 go build -o build/gopath/bin/darwin_amd64/amazon-eks-pod-identity-webhook $(GO_LDFLAGS) $V $T
-endif
-	GOOS=linux GOARCH=amd64 go build -o build/gopath/bin/linux_amd64/amazon-eks-pod-identity-webhook $(GO_LDFLAGS) $V $T
+	hack/install.sh
 
 # Generic make
-REGISTRY_ID?=602401143452
+REGISTRY?=public.ecr.aws
```

Comment:
Do we already push this image to this repo?

Reply:
this is just a placeholder for override, for local docker build, it doesn’t matter

When an external user wants to build his own image and push it, he can do:
IMAGE=xxx make image
then do a docker pu